src/main/java/use_case/signup/student_signup/StudentSignupUserDataAccessInterface.java

```diff
@@ -12,19 +12,19 @@ public interface StudentSignupUserDataAccessInterface {
      * @param username the username to look for
      * @return true if a user with the given username exists; false otherwise
      */
-    boolean existsByName(String username);
+    boolean existsByNameStudent(String username);
 
     /**
      * Checks if the given email exists.
      * @param email the email to look for
      * @return true if a user with the given email exists; false otherwise
      */
-    boolean existsByEmail(String email);
+    boolean existsByEmailStudent(String email);
 
     /**
      * Saves the student user.
      * @param user the student user to save
      */
-    void saveStudent(User user);
+    void saveStudent(Student user);
 
```

Comment:
no real complaints, a lot of these changes are minor (but necessary) and seem to be done cleanly.

Reply:
Thanks :)